app/helpers/application_helper.rb

```diff
@@ -94,12 +94,14 @@ def javascript_doc_ready(&block)
     javascript_tag("$(document).ready(function(){#{content}});")
   end
 
-  # google maps
-  def javascript_google_maps
-    if configatron.has_key(:google_maps_api_key)
-      api_key = configatron.google_maps_api_key
-      javascript_include_tag("https://maps.googleapis.com/maps/api/js?key=#{api_key}&v=3")
-    end
+  def google_maps_key_missing?
+    !configatron.has_key?(:google_maps_api_key)
+  end
+
+  def google_maps_js
+    return "" if google_maps_key_missing? && !offline?
```

Comment:
Is this right? Worth a comment. It seems to return an (invalid) maps URL if the key is missing and you **are** offline.

Reply:
Rats. That should be an OR! I'll fix.

Reply:
I blame the virus...